cyhy/mailer/cli.py

```diff
@@ -761,6 +793,27 @@ def send_cyhy_reports(db, batch_size, ses_client, cyhy_report_dir, dry_run=False
             if not to_emails:
                 continue
 
+            # Grab the email address of the appropriate CSA.
+            csa_email_address = None
+            if csa_emails is not None:
+                try:
+                    state = request["agency"]["location"]["state"]
+                except KeyError:
+                    logging.warn(
+                        f"State or territory information unavailable for agency with ID {id}.  As a result this CyHy report cannot be sent to the corresponding CSA."
+                    )
+                else:
+                    try:
+                        csa_email_address = csa_emails[state]
+                    except KeyError:
+                        logging.warn(
+                            f"State or territory {state} associated with agency with ID {id} does not correspond to a CSA region.  As a result this CyHy report cannot be sent to the corresponding CSA."
+                        )
+                    else:
+                        logging.debug(
+                            f"BCCing report for agency with ID {id} to {csa_email_address} because the agency is associated with the state or territory {state}."
```

Comment:
Even though our request document structure still calls everyone an "agency", I think at this point, we should be shifting to either "entity" or "organization" (my preference is "entity"):
```suggestion
                        f"State or territory information unavailable for entity with ID {id}.  As a result this CyHy report cannot be sent to the corresponding CSA."
                    )
                else:
                    try:
                        csa_email_address = csa_emails[state]
                    except KeyError:
                        logging.warn(
                            f"State or territory {state} associated with entity with ID {id} does not correspond to a CSA region.  As a result this CyHy report cannot be sent to the corresponding CSA."
                        )
                    else:
                        logging.debug(
                            f"BCCing report for entity with ID {id} to {csa_email_address} because the entity is associated with the state or territory {state}."
```

Though it might make more sense to log an issue and take care of this wholesale in this repo?

Reply:
Please see commits 0e0231f and 840fd56.

Reply:
This request caused me to notice [another issue](https://github.com/cisagov/cyhy-mailer/issues/102), which I have assigned to you.

Reply:
Lovely! 👍 👍